app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -31,14 +31,14 @@ def create_metadata(file_set_params = {})
 
       # Called from FileSetsController, AttachFilesToWorkJob, ImportURLJob, IngestLocalFileJob
       # @param [File, ActionDigest::HTTP::UploadedFile, Tempfile] file the file uploaded by the user.
-      # @param [String] relation ('original_file')
-      # @param [Boolean] asynchronous (true) set to false if you don't want to launch a new background job.
-      def create_content(file, relation = 'original_file', asynchronous = true)
+      # @param [Symbol, #to_sym] relation
+      # @return [Boolean] true on success, false otherwise
+      def create_content(file, relation = :original_file)
```

Comment:
Should there be a deprecation of the third argument?  Why did the type of the second argument change?